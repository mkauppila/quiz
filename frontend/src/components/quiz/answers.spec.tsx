import * as React from 'react'
import { shallow } from 'enzyme'
import Answers from './answers'

const answers = [
  {
    answer: '8 seasons'
  },
  {
    answer: '14 seasons'
  },
  {
    answer: '12 seasons'
  },
  {
    answer: '6 seasons'
  },
]

describe('anwers component', () => {
  it('a', () => {
    const component = shallow(<Answers answers={answers} action={() => {}} />)
    expect(component).toMatchSnapshot()
  })
})