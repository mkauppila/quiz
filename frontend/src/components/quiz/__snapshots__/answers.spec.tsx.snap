// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`anwers component a 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    answers={
      Array [
        Object {
          "answer": "8 seasons",
        },
        Object {
          "answer": "14 seasons",
        },
        Object {
          "answer": "12 seasons",
        },
        Object {
          "answer": "6 seasons",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <styled.ul>
        <styled.li>
          <styled.input
            name="8 seasons"
            value="8 seasons"
          />
          8 seasons
        </styled.li>
        <styled.li>
          <styled.input
            name="14 seasons"
            value="14 seasons"
          />
          14 seasons
        </styled.li>
        <styled.li>
          <styled.input
            name="12 seasons"
            value="12 seasons"
          />
          12 seasons
        </styled.li>
        <styled.li>
          <styled.input
            name="6 seasons"
            value="6 seasons"
          />
          6 seasons
        </styled.li>
        <styled.input
          value="Vastaa"
        />
      </styled.ul>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          Array [
            <styled.li>
              <styled.input
                name="8 seasons"
                value="8 seasons"
              />
              8 seasons
            </styled.li>,
            <styled.li>
              <styled.input
                name="14 seasons"
                value="14 seasons"
              />
              14 seasons
            </styled.li>,
            <styled.li>
              <styled.input
                name="12 seasons"
                value="12 seasons"
              />
              12 seasons
            </styled.li>,
            <styled.li>
              <styled.input
                name="6 seasons"
                value="6 seasons"
              />
              6 seasons
            </styled.li>,
          ],
          <styled.input
            value="Vastaa"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "8 seasons",
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <styled.input
                name="8 seasons"
                value="8 seasons"
              />,
              "8 seasons",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "name": "8 seasons",
                "value": "8 seasons",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "8 seasons",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "14 seasons",
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <styled.input
                name="14 seasons"
                value="14 seasons"
              />,
              "14 seasons",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "name": "14 seasons",
                "value": "14 seasons",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "14 seasons",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "12 seasons",
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <styled.input
                name="12 seasons"
                value="12 seasons"
              />,
              "12 seasons",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "name": "12 seasons",
                "value": "12 seasons",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "12 seasons",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "6 seasons",
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <styled.input
                name="6 seasons"
                value="6 seasons"
              />,
              "6 seasons",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "name": "6 seasons",
                "value": "6 seasons",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "6 seasons",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "value": "Vastaa",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "form",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <styled.ul>
          <styled.li>
            <styled.input
              name="8 seasons"
              value="8 seasons"
            />
            8 seasons
          </styled.li>
          <styled.li>
            <styled.input
              name="14 seasons"
              value="14 seasons"
            />
            14 seasons
          </styled.li>
          <styled.li>
            <styled.input
              name="12 seasons"
              value="12 seasons"
            />
            12 seasons
          </styled.li>
          <styled.li>
            <styled.input
              name="6 seasons"
              value="6 seasons"
            />
            6 seasons
          </styled.li>
          <styled.input
            value="Vastaa"
          />
        </styled.ul>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            Array [
              <styled.li>
                <styled.input
                  name="8 seasons"
                  value="8 seasons"
                />
                8 seasons
              </styled.li>,
              <styled.li>
                <styled.input
                  name="14 seasons"
                  value="14 seasons"
                />
                14 seasons
              </styled.li>,
              <styled.li>
                <styled.input
                  name="12 seasons"
                  value="12 seasons"
                />
                12 seasons
              </styled.li>,
              <styled.li>
                <styled.input
                  name="6 seasons"
                  value="6 seasons"
                />
                6 seasons
              </styled.li>,
            ],
            <styled.input
              value="Vastaa"
            />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "8 seasons",
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <styled.input
                  name="8 seasons"
                  value="8 seasons"
                />,
                "8 seasons",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "name": "8 seasons",
                  "value": "8 seasons",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "8 seasons",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "14 seasons",
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <styled.input
                  name="14 seasons"
                  value="14 seasons"
                />,
                "14 seasons",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "name": "14 seasons",
                  "value": "14 seasons",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "14 seasons",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "12 seasons",
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <styled.input
                  name="12 seasons"
                  value="12 seasons"
                />,
                "12 seasons",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "name": "12 seasons",
                  "value": "12 seasons",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "12 seasons",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "6 seasons",
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <styled.input
                  name="6 seasons"
                  value="6 seasons"
                />,
                "6 seasons",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "name": "6 seasons",
                  "value": "6 seasons",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "6 seasons",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "value": "Vastaa",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "form",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
